code/game/gamemodes/gang/gang.dm

```diff
@@ -254,6 +254,16 @@ GLOBAL_LIST_INIT(gang_colors_pool, list("red","orange","yellow","green","blue","
 		gang_bosses += G.bosses
 	return gang_bosses
 
+/datum/game_mode/proc/shuttle_check()
```

Comment:
oh something call its but because of fucking commit memes I have to rewrite the line